### ISSUE TITLE
Invalid JSON written to Durable Entity

### ISSUE THREAD

**Issue by 7CLewis** [Azure/azure-functions-durable-extension#2920]
### Description
Invalid JSON written to Durable Entity, overwriting valid data with a sequence of NULL characters and causing a malformed JSON error when trying to read the entity's state.

### Expected behavior
Entity state to be correctly written to Durable Entity file in Blob Container

### Actual behavior
A long sequence of NULL (\u0000) characters was written into the Blob Container file for one of our Durable Entities. This overwrote some data and caused a malformed JSON error when we called ReadEntityStateAsync in our .NET Function app. It seems that this occurred on the Azure side when it was writing the Blob Container file, since our app could not possibly have written invalid JSON to the file in that way. The entity file was approximately 14MB at the time.

### Relevant source code snippets
An invalid JSON error occurred when trying to read the Entity State with
```
...
var instance = await client.ReadEntityStateAsync<ARCustomer>(
    entityId
);
...
```

### Known workarounds
We downloaded the invalid entity file, manually removed the invalid NULL characters, and then re-uploaded (all while the app was stopped) it to fix the issue.

### App Details
- **Programming language used**: .NET 6
- **Microsoft.Azure.WebJobs.Extensions.Storage**: 5.0.1
- **Microsoft.Azure.Functions.Extensions**: 1.1.0
- **Microsoft.Azure.WebJobs.Extensions.DurableTask**: 2.9.5

### If deployed to Azure
> We have access to a lot of telemetry that can help with investigations. Please provide as much of the following information as you can to help us investigate!

- **Timeframe issue observed**: 19 Sept 2024
- **Function App name**: [can't share]
- **Function name(s)**: EternalGenerateAllExportFiles
- **Orchestration instance ID(s)**: 
  - $0e369c29e36a4dde9fee7c02aba99592
  - $13614bad1a974d509d154640ccad4686
  - $fdd76fd931e5478299ac883d689a9f61
- **Azure storage account name**: [can't share]

> If you don't want to share your Function App or storage account name GitHub, please at least share the orchestration instance ID. Otherwise it's extremely difficult to look up information.

**Comment by cgillum** [comment]
Hi @7CLewis. It seems like the version of the DurableTask nuget package you're using is quite old, which may make it difficult to try and diagnose this problem. Can you try upgrading to the latest released version ([2.13.6](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask/2.13.6)) to see if the issue is still reproducible?

**Comment by 7CLewis** [comment]
@cgillum Alright, we've update the package and will reopen a ticket if it happens again.
